src/definitions/elements/list.less

```diff
@@ -218,6 +218,16 @@ ol.ui.list ol li,
   color: @itemLinkIconColor;
 }
 
+.ui.list .list > a.item > i.icons,
+.ui.list > a.item > i.icons {
```

Comment:
You may just move this to the above existing declaration for single icon, so it becomes

```css
/* Linked Item Icons */
.ui.list .list > a.item > i.icons,
.ui.list > a.item > i.icons,
.ui.list .list > a.item > i.icon,
.ui.list > a.item > i.icon {
  color: @itemLinkIconColor;
}
```